src/gui/guiFormSpecMenu.cpp

```diff
@@ -2191,22 +2218,34 @@ void GUIFormSpecMenu::regenerateGui(v2u32 screensize)
 		m_font = g_fontengine->getFont();
 		m_btn_height = font_line_height(m_font) * 0.875;
 		DesiredRect = core::rect<s32>(
-			(s32)((f32)mydata.screensize.X * mydata.offset.X) - (s32)(mydata.anchor.X * 580.0),
-			(s32)((f32)mydata.screensize.Y * mydata.offset.Y) - (s32)(mydata.anchor.Y * 300.0),
-			(s32)((f32)mydata.screensize.X * mydata.offset.X) + (s32)((1.0 - mydata.anchor.X) * 580.0),
-			(s32)((f32)mydata.screensize.Y * mydata.offset.Y) + (s32)((1.0 - mydata.anchor.Y) * 300.0)
+				(s32) ((f32) mydata.screensize.X * mydata.offset.X) -
+						(s32) (mydata.anchor.X * 580.0),
+				(s32) ((f32) mydata.screensize.Y * mydata.offset.Y) -
+						(s32) (mydata.anchor.Y * 300.0),
+				(s32) ((f32) mydata.screensize.X * mydata.offset.X) +
+						(s32) ((1.0 - mydata.anchor.X) * 580.0),
+				(s32) ((f32) mydata.screensize.Y * mydata.offset.Y) +
+						(s32) ((1.0 - mydata.anchor.Y) * 300.0)
 		);
 	}
 	recalculateAbsolutePosition(false);
 	mydata.basepos = getBasePos();
 	m_tooltip_element->setOverrideFont(m_font);
 
-	gui::IGUISkin* skin = Environment->getSkin();
+	gui::IGUISkin *skin = Environment->getSkin();
 	sanity_check(skin);
 	gui::IGUIFont *old_font = skin->getFont();
 	skin->setFont(m_font);
 
 	pos_offset = v2s32();
+
+	if (enable_prepends) {
+		std::vector<std::string> prependElements = split(m_formspec_prepend, ']');
+		for (const auto& element : prependElements) {
```

Comment:
`for (const std::string &element : prepend_elements)`
Bracket removal possible, corrected the reference position and the vector name to our guidelines.